### PR TITLE
Atom edited changes during translations

### PR DESCRIPTION
**Pull request by smoe**
Still got quite a bit ahead of us.

### CODE REVIEW

**Comment by smoe** [comment]
No idea why this fails.

File: docs/README.adoc [hunk 1]
```diff
@@ -137,9 +137,9 @@ the LinuxCNC community.
 
 === Drawings from QCAD
 
-- Save as .dxf to the same directory as the image
-- Set line thickness to 0.50mm
-- Export the  image as 480 x 480 Resolution Auto .png
+- Save as .dxf to the same directory as the image.
+- Set line thickness to 0.50 mm.
```
Comment (hansu) [inline docs/README.adoc:141]: What about the thin spaces? Or do we want to insert them all with one commit?

File: docs/src/lathe/lathe-user.adoc [hunk 2]
```diff
@@ -27,13 +27,13 @@ To set up AXIS for Lathe Mode.
 ----
 [DISPLAY]
 
-# Tell the Axis Display our machine is a lathe.
+# Tell the AXIS display (VCP) our machine is a lathe.
```
Comment (hansu) [inline docs/src/lathe/lathe-user.adoc:30]: Why "(VCP)" ?

File: docs/src/plasma/qtplasmac.adoc [hunk 3]
```diff
@@ -3517,9 +3613,11 @@ setp plasmac.zero-window 5
 
 Max Offset is the distance (in millimeters) away from the Z MAX_LIMIT that QtPlasmaC will allow the Z axis to travel while under machine control.
 
-The pin for adjusting this value is named plasmac.max-offset and the default value (in millimeters) is set to 5. To change this value, add the pin and the required value to the custom.hal file. It is not recommended to use values less than 5mm as offset overrun may cause unforeseen issues.
+The pin for adjusting this value is named plasmac.max-offset and the default value (in millimeters) is set to 5.
+To change this value, add the pin and the required value to the custom.hal file.
+It is not recommended to use values less than 5mm as offset overrun may cause unforeseen issues.
 
-The following example would set the distance from Z MAX_LIMIT to 10mm:
+The following example would set the distance from Z MAX_LIMIT to 10 mm:
```
Comment (hansu) [inline docs/src/plasma/qtplasmac.adoc:3620]: same here

File: docs/src/lathe/lathe-user.adoc (hunk 2 above)
Comment (smoe) [reply docs/src/lathe/lathe-user.adoc:30]: Isn't AXIS the virtual control panel? I found "Display" misleading.

File: docs/README.adoc (hunk 1 above)
Comment (smoe) [reply docs/README.adoc:141]: I had not thought about the thin spaces. To have a dedicated concerted action on all the units sounds good.

File: docs/src/lathe/lathe-user.adoc (hunk 2 above)
Comment (hansu) [reply docs/src/lathe/lathe-user.adoc:30]: According to the docs, VCPs are the custom panels you can add to the GUI.
Yeah display sound not so good for this. What abut " # Tell the AXIS GUI our machine is a lathe. " ?

File: docs/src/gui/gmoccapy.adoc [hunk 4]
```diff
@@ -14,34 +14,34 @@
...
-'GMOCCAPY' is a GUI for LinuxCNC, designed to be used with a touch screen,
+GMOCCAPY is a GUI for LinuxCNC, designed to be used with a touch screen,
 but can also be used on normal screens with a mouse or hardware buttons and MPG
 wheels, as it presents HAL Pins for the most common needs. Please find more
 information in the following.
 
 It offers the possibility to display up to 9 axis, support a lathe mode for
 normal and back tool lathe and can be adapted to nearly every need, because
-gmoccapy supports embedded tabs and side panels.
+GMOCCAPY supports embedded tabs and side panels.
```
Comment (hansu) [inline docs/src/gui/gmoccapy.adoc:24]: Why uppercase letters here?

Not sure which is @gmoccapy's preferred spelling...

**Comment by smoe** [comment]
No reaction from @gmoccapy, yet. I suggest to go ahead with the all-caps (or to something else unique like GMoccaPy) throughout the documentation while we are all iterating over it during the translations and then change it to whatever we are instructed to.

**Comment by hansu** [comment]
> No reaction from @gmoccapy, yet. I suggest to go ahead with the all-caps (or to something else unique like GMoccaPy) throughout the documentation while we are all iterating over it during the translations and then change it to whatever we are instructed to.

According to Norbert, 'GMOCCAPY' is fine :wink:

**Comment by hansu** [comment]
@smoe Please rebase to resolve the conflicts

**Comment by jepler** [comment]
Can you re-word or explain what "Atom" means in the PR title?

**Comment by hansu** [comment]
I guess the [text editor named Atom](https://en.wikipedia.org/wiki/Atom_(text_editor)) is meant here.

**Comment by petterreinholdtsen** [comment]
Are all the issues brought up in the comments handled now?

**Comment by hansu** [comment]
Yes.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
>  No idea why this fails.

Perhaps this one:

asciidoc: ERROR: qtplasmac.adoc: line 3376: illegal style name: %header

Time to stop trusting your editor?

-- 
Happy hacking
Petter Reinholdtsen

**Pull request merged**